db/seeds.rb

```diff
@@ -5,3 +5,60 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+p = Path.create!(
+      name: 'Full Stack Developer',
+      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Urna rutrum tellus rutrum feugiat cursus ipsum eget nulla. Auctor imperdiet odio vulputate porta arcu nisl purus lobortis eget. Cras ac eu ut urna purus, varius nisl tincidunt velit. Dictumst sed sit massa mattis facilisi molestie sed quisque. Enim aenean nec semper a, tempor eget pharetra tristique viverra.',
+      requirement: 'beginner',
+      language: 'ruby javascript html css rails'
+      )
+
+p "Created the PATH 'Full Stack Developer'!"
+
+12.times do
+  Step.create!(
+    path_id: p.id,
+    url: 'https://www.codecademy.com/courses/react-101/lessons/react-components-advanced-jsx/exercises/render-multiline-jsx',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Urna rutrum tellus rutrum feugiat cursus ipsum eget nulla. Auctor imperdiet odio vulputate porta arcu nisl purus lobortis eget. Cras ac eu ut urna purus, varius nisl tincidunt velit. ',
+    title: 'Learn Flexbox',
+    step_type: 'module',
+    duration: 45
+    )
+end
+
+p "Added 12 identical STEPs to the path!"
+
+lg = LearningGroup.create!(
+  path_id: p.id,
+  slack_channel: '#clp'
+  )
+
+p "Initialized a LEARNING_GROUP, associated to the path with the slack channel '#clp'"
+
+Step.all.each_with_index do |step, i|
+  StepProgress.create!(
+    user_id: User.first.id,
```

Comment:
This means that in order to run this seed we need to have created a user first (through the web-interface). Is there a way to create the user in the seed? probably not, huh?
Also: i am currently changing the user model to require a name. Will this break stuff, when we merge it, because suddenly all existing users are not valid records anymore? Probably it will, right? And the solution would be to add a name to existing users in the rails console after merging my PR?

Reply:
Yes, you have to create a user first manually on the sign up page. I don't know whether it's possible to do it in the seeds file because every user has an encrypted password and I don't know how the encryption works.

I think the validation checks an entry only when it's added to the db so I think the validation won't cause any errors when you merge your pull request, although we will run into issues later (when we try to use the user's name) so it probably makes sense to add a name for the user(s) we've already created manually in the console.

Reply:
So Iva told me that it's possible to add a user in the seed which I now did.
I've created a user with a name so we have to run the migration for the name first.